speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/conformationmenu.cpp

```diff
@@ -0,0 +1,84 @@
+/***************************************************************************
+
+    file                 : savemenu.cpp
+    created              : Tue Mar 18 10:54:12 CET 2022
+    copyright            : (C) 2022 by Maik Vink
+    email                : torcs@free.fr
+    version              : $Id: savemenu.cpp $
+
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
```

Comment:
Remove this, we don't credit our files like this. Also, files created by us should be UpperCamelCase, so rename to ConfirmationMenu.cpp

Also run the linter on this file, because it has a lot of code style issues.